### ISSUE TITLE
Cannot use all the GPU resource

### ISSUE THREAD

**Issue by hutianyu2006** [pkhungurn/talking-head-anime-demo#24]
When I an running this project with my GTX 960M, I found out that the GPU resource haven't been used up , but it is still very laggy. How can I run it with full speed?

**Comment by Northurland** [comment]
Me too, but I have a Quadro P2000 instead :(

**Comment by Northurland** [comment]
> 
> 
> Me too, but I have a Quadro P2000 instead :(

My extra information:
Python version: 3.9.0
PyTorch version: 1.7.1
Cuda version: 11.0.228 
Used operating system: Windows 10 (1909)
CPU: Intel i7-9750H

The console wasn't actually reporting errors, but it does give warnings about implicit conversion to integers.

**Comment by dragonmeteor** [comment]
See https://github.com/pkhungurn/talking-head-anime-demo/issues/10

TL;DR: Windows does not report the right number, and your GPU was probably used to its maximum capacity.
